aioesphomeapi/api.proto

```diff
@@ -1166,12 +1183,13 @@ message BluetoothLEAdvertisementResponse {
   option (no_delay) = true;
 
   uint64 address = 1;
-  bytes name = 2;
+  string name = 2;
```

Comment:
This is another breaking change - but super convenient having the name coerced to a string.

Reply:
https://github.com/esphome/aioesphomeapi/pull/339